flashinfer/fused_moe/dist_aware/da_profile.py

```
@@ -0,0 +1,1859 @@
+        return False
+
+    _bundle_loaded = True
+    da_state.BUNDLE_LOADED_CONTEXTS.add(da_context)
+    try:
+        hidden_size = int(hidden_states.shape[-1])
+        if hidden_states.dtype == torch.uint8:
+            hidden_size *= 2
+        import pickle
+
+        with open(path, "rb") as bundle_file:
+            bundle = pickle.load(bundle_file)
```

Ruff (0.15.21)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)